pkg/apis/nfd/validate/validate.go

```diff
@@ -79,6 +80,13 @@ func Template(labelsTemplate string) []error {
 	return validationErr
 }
 
+func sortErrors(errs []error) []error {
```

Comment:
shouldn't we just invoke this func in tests? what's the value in sorting errors by char len?

Reply:
It doesn't sort by length but alphabetically.

Sorting can be done in tests, yes. I moved it there.